File: server-scala-play-server/app/model/FlagThreshold.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for FlagThreshold.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class FlagThreshold(
  id: Option[Long],
  active: Option[Boolean],
  valid: Option[Boolean],
  threshold: Option[Long],
  flagableObjectType: Option[String],
  application: Option[Application]
)

object FlagThreshold {
  implicit lazy val flagThresholdJsonFormat: Format[FlagThreshold] = Json.format[FlagThreshold]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/FlagThreshold.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class FlagThreshold 
{
    public long Id { get; set; }
    public bool Active { get; set; }
    public bool Valid { get; set; }
    public long Threshold { get; set; }
    public string FlagableObjectType { get; set; }
    public Application Application { get; set; }
}


